includes/languages/english.php

```diff
@@ -570,7 +570,7 @@ function zen_date_raw($date, $reverse = false) {
 
   define('DB_ERROR_NOT_CONNECTED', 'Error - Could not connect to Database');
   define('ERROR_DATABASE_MAINTENANCE_NEEDED', '<a href="https://docs.zen-cart.com/user/troubleshooting/error_71_maintenance_required/" rel="noopener" target="_blank">ERROR 0071 There appears to be a problem with the database. Maintenance is required.</a>');
-
+  
```

Comment:
Sorry to be picky, but your last update still has the lang file updated by adding spaces.

Reply:
Can you fix this for me?  I'm away from my home office.

Reply:
this should be fixed now.